static/app/views/performance/landing/widgets/widgets/trendsWidget.tsx

```diff
@@ -86,7 +86,7 @@ export function TrendsWidget(props: Props) {
         transform: transformTrendsDiscover,
       },
     };
-  }, [_eventView, trendChangeType]);
+  }, [eventView.query, eventView.fields, trendChangeType]);
```

Comment:
I may be wrong but since `eventView` is derived from the location each time, wouldn't this cause `useMemo` to re-fire on every location change regardless if the `fields` changed or not?

Reply:
I still have to write some tests to ensure memoization is working correctly, likely as part of the batch query handler PR, but I _think_ this should be fine for now. `eventView.query` is a string, which can be compared as a primitive, which means it shouldn't cause it to refire if eventView is recreated, which is what was happening before (sorting the table caused widgets to reload... not ideal)